Pop-Grupa-A-Frontend/src/components/computationCockpit/taskList/TaskList.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+
+class TaskList extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            listitems: [],
+            userId: 1,
+            isLoading: false
+        };
+    }
+
+    getTasks() {
+        fetch("http://localhost:5000/task/" + this.state.userId, {
+            crossDomain: true,
+            method: 'get',
+        })
+            .then(response => response.json())
+            .then(response => {
+                this.setState({
+                    listitems: response
+                })
+            });
+    }
+
+    componentDidMount() {
+        this.getTasks();
+    }
+
+    componentWillUpdate() {
```

Comment:
niekontrolujecie wywołań ten metody.  W getTasks pobieracie taski i zmieniacie stan co triggeruje rerender komponentu, wchodzi do tej metody ,widzi ze komponent bedzie sie update'owal i znowu pobiera taski.
![image](https://user-images.githubusercontent.com/26284998/67710648-9af07e00-f9c0-11e9-8bc4-ba5e08d0d19e.png)

Możecie użyć tej w celu kontroli zmiany stanu/przekazywanych propsów dla komponentu.
`componentDidUpdate(prevProps) { `
 `    // Typowy sposób użycia (nie zapomnij porównać właściwości): `
`     if (this.props.userID !== prevProps.userID) { ` 
 `         this.fetchData(this.props.userID);  `
 `     }  `
`}`

Reply:
To jest do chłopaków od Computation cockpit w sumie. My tylko to od nich zmergowaliśmy aby widok móc zrobić

Reply:
Poprawiłem, możesz @kolendomichal sprawdzić czy może być tak jak teraz jest

Reply:
Ok ,myślałem,że wasze. Teraz jest już git.